runtime/compiler/trj9/codegen/J9AheadOfTimeCompile.cpp

```diff
@@ -936,6 +936,41 @@ J9::AheadOfTimeCompile::dumpRelocationData()
                   }
                }
             break;
+         case TR_DebugCounter:
+            cursor ++;
+            if (is64BitTarget)
+               {
+               cursor += 4;     // padding
+               ep1 = cursor;    // inlinedSiteIndex
+               ep2 = cursor+8;  // bcIndex
+               ep3 = cursor+16; // keyOfNameString
+               ep4 = cursor+24; // delta
+               ep5 = cursor+40; // staticDelta
```

Comment:
Can we not use `offsetof(TR_RelocationRecordDebugCounterBinaryTemplate, _staticDelta)` here and in nearby places? This seems not very robust to changes.

Reply:
> This seems not very robust to changes.
Yeah, the entire file follows this paradigm. At some point when I get time I want to clean up the AOT store-to-scc code (the load-from-scc code is already properly refactored). For now though, I'll do what you suggest with this bit of code.

Reply:
Heh I'm not easily able to change the code to use `offsetof(TR_RelocationRecordDebugCounterBinaryTemplate, _staticDelta)`; I'd have to do something like `cursor+offsetof(TR_RelocationRecordDebugCounterBinaryTemplate, _staticDelta)-sizeof(TR_RelocationRecordBinaryTemplate)` which is uglier and lacks robustness in a diff way (since changing the inheritance graph would cause problems again). I'll leave it as is for now.